_posts/2021-08-24-devs-be-like.md

```diff
@@ -0,0 +1,70 @@
+
+![Programmer](/img/devs-be-like/programmer.png)
+
+*Photo by <a href="https://www.pexels.com/@cottonbro" target="_blank">cottonbro</a> from Pexels*
+
+I'm sure people like that exist but I'm not sure I know any personally. 
+
+## Education
+
+I began my journey to be a developer in my mid-20s when I started studying computer science at a university. I had absolutely no experience with programming (well, I used to have a MySpace page and I updated the theme probably twice a week, so I had tinkered with css a lot). In the beginning of my university studies I was very nervous about fitting in and not being "outed as a hoax" because I thought everybody else knew how to code before coming to university. At least in Finland the stereotype about knowing how to code before going to a school to learn how to code sits very tight. I think very few of those students who I spent my first year with actually knew any coding. You wouldn't expect a person starting at a medical school to already know how to do surgery or to make diagnosis so why should computer science students know how to code?
+
+The stereotype about self-learned developers can be motivating to some. And I think it is great that you can find work as a developer without any formal education. If a teenager spends their time learning programming, that's great, especially since there are probably worse ways for a teenager to spend their time. But the industry has possibly missed great developers because someone didn't have the time or the means as a kid to learn programming. Not everybody has a good enough computer at home that allows them to learn and develop their skills. Or they might have a computer at home, but they need to share it with the rest of the family. Especially if the decision about one's future career has to be made around your teenage years, the current stereotypes about developers surely do not encourage to seek the profession if you do not have had the opportunity to learn.
+
+![Child learning](/img/devs-be-like/kid-learning.png)
+
+*Photo by <a href="https://unsplash.com/@hindawi" target="_blank">hindawi</a> on Unsplash*
+
+I'm not saying there's anything wrong with teaching kids some programming or being interested and learning stuff by yourself. However, people should also be taught that you can start learning programming at any age. Going to a school to learn computer science or information technology without any previous knowledge is totally fine. It's called a _school_ for a reason, right?
+
+## Getting a job
+
+Getting your first job as a developer can either be easy as heck or the most stressful thing in your entire year. When I was in university, my friends who had time, energy, and interest to do serious hobby projects got job offers at job fairs just by showing what they can do. The rest of us were left feeling (at least I was) overwhelmed and a little scared. 
```

Comment:
This is honest and interesting to many students. Also, it was nice to think back and memorize when reading this. 👍